modules/configuring-localnet-switched-topology.adoc

```diff
@@ -6,50 +6,48 @@
 [id="configuration-localnet-switched-topology_{context}"]
 = Configuration for a localnet switched topology
 
-// To accommodate a link to the NMstate Operator, the content in this module
-// is split with tags. The tag includes don't pull in the module header above.
-
-// tag::localnet-intro[]
+[role="_abstract"]
 The switched `localnet` topology interconnects the workloads created as Network Attachment Definitions (NADs) through a cluster-wide logical switch to a physical network.
-// end::localnet-intro[]
 
-// tag::localnet-content[]
-You must map a secondary network to the ovs-bridge to use it as an OVN-Kubernetes secondary network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with Open vSwitch (OVS).
+You must map a secondary network to the OVS bridge to use it as an OVN-Kubernetes secondary network. Bridge mappings allow network traffic to reach the physical network. A bridge mapping associates a physical network name, also known as an interface label, to a bridge created with Open vSwitch (OVS).
 
 You can create an `NodeNetworkConfigurationPolicy` (NNCP) object, part of the `nmstate.io/v1` API group, to declaratively create the mapping. This API is provided by the NMState Operator. By using this API you can apply the bridge mapping to nodes that match your specified `nodeSelector` expression, such as `node-role.kubernetes.io/worker: ''`. With this declarative approach, the NMState Operator applies secondary network configuration to all nodes specified by the node selector automatically and transparently.
 
 When attaching a secondary network, you can either use the existing `br-ex` bridge or create a new bridge. Which approach to use depends on your specific network infrastructure. Consider the following approaches:
 
-- If your nodes include only a single network interface, you must use the existing bridge. This network interface is owned and managed by OVN-Kubernetes and you must not remove it from the `br-ex` bridge or alter the interface configuration. If you remove or alter the network interface, your cluster network will stop working correctly.
+- If your nodes include only a single network interface, you must use the existing bridge. This network interface is owned and managed by OVN-Kubernetes and you must not remove it from the `br-ex` bridge or alter the interface configuration. If you remove or alter the network interface, your cluster network stops working correctly.
 - If your nodes include several network interfaces, you can attach a different network interface to a new bridge, and use that for your secondary network. This approach provides for traffic isolation from your primary cluster network.
 
-The `localnet1` network is mapped to the `br-ex` bridge in the following example:
+The `localnet1` network is mapped to the `br-ex` bridge in the following sharing a bridge example:
```

Comment:
```suggestion
The `localnet1` network is mapped to the `br-ex` bridge in the following sharing-a-bridge example:
```
or
The `localnet1` network is mapped to the `br-ex` bridge in the following example where the bridge is shared: